README.md

```diff
@@ -58,6 +59,25 @@ epsagon.Label('items_in_cart', items_in_cart)
 
 Valid types are `string`, `bool`, `int` and `float`.
 
+
+### Custom Errors
+
+You can set a trace as an error (although handled correctly) to get an alert or just follow it on the dashboard.
+
+Add the following call inside your code:
+```go
+epsagon.Error("My custom error")
+# Or manually add an error
+epsagon.Error(errors.New("My custom error"))
+```
+
+You can also set a tracer as an error with a custom error type, which you can use later on the dashboard.
+```go
+epsagon.ErrorWithType("My custom error", "Custom Error Type")
```

Comment:
epsagon.TypeError?
epsagon.TypedError?
epsagon.TError?

dunno, I don't really like the name